selectrum.el

```diff
@@ -695,6 +740,31 @@ PRED defaults to `minibuffer-completion-predicate'."
       ('current/matches (format "%-6s " (format "%d/%d" current total)))
       (_                ""))))
 
+(defvar display-line-numbers)
+(defun selectrum--get-display-window ()
+  "Get candidate display window.
+
+Window will be created by `selectrum-display-action'."
+  (let ((buf (or (get-buffer selectrum--candidates-buffer)
+                 (with-current-buffer
+                     (get-buffer-create selectrum--candidates-buffer)
```

Comment:
Another approach would be to

```elisp
(when-let ((buf (get-buffer selectrum--candidates-buffer)))
  (kill-buffer buf))
(let ((buf (with-current-buffer (get-buffer-create selectrum--candidates-buffer)
              ...))
  ...)
```
which has the advantage/disadvantage of if the user accidentally/intentionally changes something buffer-local about the display buffer, the changes will not persist to other sessions.

Reply:
The buffer and its window gets reused when it already exists, see later where `(get-buffer-window buf 'visible)` is used. I would like to keep it that way for now when you are okay with that. I plan working on another PR where the buffer creation process will change a bit to allow more customization of the formatting of the candidates buffer.

Reply:
Gotcha, that makes sense. Sounds like you're doing it the right way for the intended functionality.